docs/README.md

```diff
@@ -191,6 +191,7 @@ render() {
 | `rootTag` | `string` | :no_entry_sign: | `'div'` | Tag to be rendered in the root of the slider |
 | `sliderFrameTag` | `string` | :no_entry_sign: | `'ul'` | Tag to be rendered in the slider frame element |
 | `threshold` | `number` | :no_entry_sign: | `50` | Minimum of pixels to drag until the slider change the `currentSlide` |
+| `navigationStep`| `number | 'page'` | :no_entry_sign: | `'page'` | How many elements should pass when you click in the arrows to navigate |
```

Comment:
```suggestion
| `navigationStep`| `number` | :no_entry_sign: | page | How many elements should pass when you click in the arrows to navigate |
```

Reply:
I can't remove the `page` option

Reply:
but I didn't remove it, it already exists in the fourth column, you just added it more than once without need.